### PR TITLE
Bug with showing time, ending on 1.

### PR DESCRIPTION
**Pull request by relo-san**
Bug with showing time, ending on 1 (21, 31 ... 91 etc).

When the difference in the 21 days between the dates we get:
Before: "через 1 день", "1 день тому".
After: "через 21 день", "21 день тому".

### CODE REVIEW

File: Resources/translations/time.uk.xliff [hunk 1]
```diff
@@ -4,27 +4,27 @@
         <body>
             <trans-unit id="1">
                 <source>diff.ago.year</source>
-                <target>1 рiк тому|%count% роки тому|%count% рокiв тому</target>
+                <target>%count% рiк тому|%count% роки тому|%count% рокiв тому</target>
```
Comment (pilot) [inline Resources/translations/time.uk.xliff]: this is wrong, it can be `10 рiк тому` only `1 рiк тому` same for others

Comment (relo-san) [inline Resources/translations/time.uk.xliff]: Wtf? It cannot be «10 рiк тому», only «10 рокiв тому».

Comment (akovalyov) [inline Resources/translations/time.uk.xliff]: @pilot you can check [russian locale](https://github.com/KnpLabs/KnpTimeBundle/blob/master/Resources/translations/time.ru.xliff#L7) i.e.

Comment (pilot) [inline Resources/translations/time.uk.xliff]: yes, for russian also not correct
- `1 рiк тому` is equal to `<target>1 рiк тому| ... </target>`
- `10 рокiв тому` is equal `<target> .. | .. |%count% рокiв тому</target>`

so what's wrong in currection notation?

Comment (relo-san) [inline Resources/translations/time.uk.xliff]: >> so what's wrong in currection notation?
How to get «21 рік тому»? Your suggestions?

Comment (pilot) [inline Resources/translations/time.uk.xliff]: yes, agreed for such case

Comment (akovalyov) [inline Resources/translations/time.uk.xliff]: @relo-san 's modifications are correct. The bug existed and it is fixed via this diff.

**Comment by akovalyov** [comment]
@relo-san thank you.

**Pull request merged**